daemon/algod/api/server/v2/handlers.go

```diff
@@ -450,6 +649,7 @@ func (v2 *Handlers) GetStatus(ctx echo.Context) error {
 		CatchpointVerifiedAccounts:  &stat.CatchpointCatchupVerifiedAccounts,
 		CatchpointTotalBlocks:       &stat.CatchpointCatchupTotalBlocks,
 		CatchpointAcquiredBlocks:    &stat.CatchpointCatchupAcquiredBlocks,
+		MaxAccountsApiResults:       maxAcctResults,
```

Comment:
It doesn't feel right that we return that in status.. It's more of a configuration, since it'a always a static value.
Another point is that I think it'd better to "hide" it anyway : 
- a client that would be able to support using it, would also be able to process the response error code.
- a client that wouldn't be able to support it, wouldn't get much from this value being reported.

Reply:
Removed